File: template/CodeSmith/CodeGenerator/01-Templates/02-Result/Presentation/Typings/app/dbo/userTeamRole/model/userTeamRole-query.ts
import { QueryParameter } from '../../../../util';

/**
 * 查询参数
 */
export class UserTeamRoleQuery extends QueryParameter {
    /**
     * 
     */
    id;
    /**
     * 
     */
    userID;
    /**
     * 
     */
    teamID;
    /**
     * 
     */
    isAdministrator;
}

File: template/CodeSmith/CodeGenerator/01-Templates/02-Result/Services/Queries/dbo/UserTeamRoleQuery.cs
using System;
using System.ComponentModel.DataAnnotations;
using Ding;
using Ding.Datas.Queries;

namespace GitDC.Service.Queries.dbo {
    /// <summary>
    /// 查询参数
    /// </summary>
    public class UserTeamRoleQuery : QueryParameter {
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public Guid? Id { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public long? UserID { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public long? TeamID { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public bool? IsAdministrator { get; set; }
    }
}